src/tk/wurst_client/mods/WeatherMod.java

```diff
@@ -0,0 +1,63 @@
+package tk.wurst_client.mods;
+
+import tk.wurst_client.mods.Mod.Bypasses;
+import tk.wurst_client.mods.Mod.Category;
+import tk.wurst_client.mods.Mod.Info;
+import tk.wurst_client.navigator.settings.CheckboxSetting;
+import tk.wurst_client.navigator.settings.SliderSetting;
+import tk.wurst_client.navigator.settings.SliderSetting.ValueDisplay;
+
+@Info(category = Category.RENDER,
+	description = "Changes the world wheather and time in the client side.",
```

Comment:
I have fixed that. See: https://github.com/Wurst-Imperium/Wurst-MC-1.10/blob/master/src/tk/wurst_client/mods/NoWeatherMod.java#L11